server/controllers/teamController/team/index.js

```diff
@@ -0,0 +1,103 @@
+const asyncHandler = require("express-async-handler");
+const { validationResult } = require("express-validator");
+const { Team } = require("../../../models/Team");
+
+/**
+ * Create a new Team resource and store in the database.
+ * @route POST /team
+ * @returns {Object} A message and payload
+ */
+exports.createTeam = asyncHandler(async (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) return next(errors);
+
+  const { name } = req.body;
+  const team = await Team.create({
+    name,
+    owner: req.user.id,
+  });
+
+  return res.status(200).json({
+    message: "Team created successfully!",
+    payload: team,
```

Comment:
Can we change this to be namespaced under team? so `{ message: "Team created successfully!",    team: team}` to maintain consistency with `boards`

Reply:
With how teams context is setup the actions accept payloads conventionally this makes more sense to pass the payload to the reducer.